fall18/p2/README.md

```diff
@@ -1,3 +1,149 @@
 # Project 2
 
-More details coming soon.
+In this project, you'll learn about creating and calling functions, arguments, return values and math module. To start, download these files:
+
+* [main.py](https://raw.githubusercontent.com/shenghaozou/taDebug/master/main.py)
+* [project.py](https://raw.githubusercontent.com/shenghaozou/taDebug/master/project.py)
+* [test.py](https://raw.githubusercontent.com/shenghaozou/taDebug/master/test.py)
+* [area.csv](https://raw.githubusercontent.com/shenghaozou/taDebug/master/area.csv)
+* [population.csv](https://raw.githubusercontent.com/shenghaozou/taDebug/master/population.csv)
+
+You will change main.py and hand it in. You should not change test.py, and you should not hand it in; it's only purpose is to tell you your grade in advance.
+
+After you've downloaded both files to the same directory, open your terminal, navigate to that directory, and run the following:
```

Comment:
Change "both files" to "all the above 5 files" since there are more than two files.